src/error.rs

```diff
@@ -291,6 +291,13 @@ pub enum ParseError {
     ),
     /// Unbound type variable
     UnboundTypeVariables(Vec<Ident>, RawSpan),
+    /// Illegal record literal in the uniterm syntax (see RFC002). In practice, this is a record
+    /// mixing a tail and non-type constructs.
```

Comment:
Make `RFC002` be a link.

I don't understand what “mixing a tail and non-type constructs” means. But probably this would be cleared out by an example.

Reply:
A "non-type construct" is anything that wasn't allowed in the original syntax inside a record type, but is allowed in a record literal. Typically a field definition `{foo = 1}`. In the RFC we discussed that it isn't immediatly obvious what semantics to give to `{foo : Num = 1; a}` (either as a type or as a term), so we planned to forbid using a polymorphic tail `; a` for anything that isn't a record type (technically, for any `t  : UniTerm` that isn't of the form `t = !r` for `r: Types` a record type in the original syntax and `! : Term + Type -> UniTerm` the embedding).

Reply:
Don't explain it to me, instead explain it to someone working in the codebase. In 6 months.